resources/uk/gov/hmcts/helm/push-helm-charts-to-git.sh

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash
+set -x
+
+CHART_DIRECTORY=${1}
+CHART_NAME=${2}
+USER_NAME=${3}
+EMAIL_ID=${4}
+VERSION=${5}
+
+RETRIES=3
+DELAY=5
+COUNT=1
+
+if [ -d "hmcts-charts" ]; then
+  rm -rf hmcts-charts
+fi
+
+while [ $COUNT -lt $RETRIES ]; do
+  if git clone https://github.com/hmcts/hmcts-charts.git; then
+    RETRIES=0
```

Comment:
will setting this to 0 in this scenario not stop it from retrying in case of git push ?